guides/common/modules/proc_configuring-project-with-keycloak-for-totp-authentication.adoc

```diff
@@ -1,14 +1,19 @@
 [id="configuring-{project-context}-with-keycloak-for-totp-authentication_{context}"]
 = Configuring {Project} with {keycloak} for TOTP authentication
 
-Use this procedure to configure {Project} to use {keycloak} as an OpenID provider for external authentication with Time-based One-time Password (TOTP).
+If you want users to authenticate with time-based one-time passwords (TOTP), configure an OTP policy for the {Project} realm in {keycloak}.
 
 .Procedure
 
 . In the {keycloak} web UI, navigate to the {Project} realm.
-. Navigate to *Authentication*, and click the *OTP Policy* tab.
-. Ensure that the *Supported Applications* field includes FreeOTP or Google Authenticator.
+. Navigate to *Authentication*.
+. On the *Policies* tab, click the *OTP Policy* tab.
+Ensure that the *Supported Applications* field includes FreeOTP or Google Authenticator.
 . Configure the OTP settings to suit your requirements.
-. Optional: If you want to use TOTP authentication as a default authentication method for all users, click the *Flows* tab, and to the right of the *OTP Form* setting, select *REQUIRED*.
-. Click the *Required Actions* tab.
-. To the right of the *Configure OTP* row, select the *Default Action* checkbox.
+. On *Required Actions* tab, enable the *Set as default action* setting for the *Configure OTP* action.
+
+ifdef::satellite[]
```

Comment:
Hmm I thought these references were previously also in the Foreman docs,
I might have missed a policy/standard though.
Up to now I always saw `ifndef::orcharhino[]` instead of `ifdef::satellite[]`, right now this makes some links show up and most of the Additional resources sections not.

Reply:
Good catch, I updated this and a couple of other ifdefs.